client/src/modules/patients/record/patient-record.html

```diff
@@ -99,6 +99,13 @@
                   <span translate>PATIENT_EDIT.EDIT_DETAILS</span>
                 </a>
 
+                <a
+                  data-method="edit"
+                  ui-sref="patientAsEmployee({ id : PatientRecordCtrl.patient.uuid })"
```

Comment:
👍 This is a good link.

Could we call it `uuid` instead of `id`?  It will help use remember that it's a string, not a number.